fecgov/fec-cms
# Feature/6390 webpack 5 fixes

## Summary

- Resolves #6390 

(Include a summary of proposed changes and connect issue below)

### Required reviewers

(Include who is required to review prior to merge. For example: One designer and two front end developer reviews are required prior to merge)

## Impacted areas of the application

General components of the application that this PR will affect:

-  

## Screenshots

Newly self-deactivating administrative fines calculator button
![image](https://github.com/user-attachments/assets/c07e52e4-5459-4d56-a339-2323a0abadaa)


## Related PRs



## How to test

- Check in the feature space, or
- For localhost
   - pull the branch
   - `npm i`
   - [`npm run build`|`npm run build-production`]
